controllers/goharbor/harborcluster/status.go

```diff
@@ -50,7 +50,7 @@ func newStatus(source *goharborv1.HarborCluster) *status {
 		cr:     source,
 		locker: &sync.Mutex{},
 		data: &goharborv1.HarborClusterStatus{
-			Status:     goharborv1.StatusUnknown,
+			Status:     goharborv1.StatusProvisioning,
```

Comment:
It seems you did not touch any overall status calculation logic in this PR, is that correct?

Reply:
Offline discussed, it's clear now.